app/client/components/payment/update/card/stripeCardInputForm.tsx

```diff
@@ -173,18 +82,26 @@ export class StripeCardInputForm extends React.Component<
     }
   };
 
-  private startCardUpdate = (navigate: NavigateFn) => async () => {
-    this.setState({ isValidating: true });
-    if (this.props.stripe && this.props.stripeSetupIntent) {
-      const createPaymentMethodResult = await this.props.stripe.createPaymentMethod(
-        "card",
-        {
-          billing_details: {
-            name: this.props.userEmail,
-            email: this.props.userEmail
-          }
+  const startCardUpdate = (navigate: NavigateFn) => async () => {
+    setIsValidating(true);
+    const cardElement = elements?.getElement(CardNumberElement);
+    if (!cardElement) {
```

Comment:
have you simulated this (by faking the condition) to test the error looks right? (and that the `return;` rather than `return null;` is legit?

Reply:
This is the error you get:

![Screenshot 2020-07-03 at 12 47 44](https://user-images.githubusercontent.com/2510683/86466411-79a01880-bd2b-11ea-97c9-c1a9e0d93bfa.png)

The return craps you out correctly yep :)